src/omv/sensors/ov7725.c

```diff
@@ -683,7 +683,7 @@ int ov7725_init(sensor_t *sensor) {
     sensor->hw_flags.jpege = 0;
     sensor->hw_flags.gs_bpp = 2;
     sensor->hw_flags.rgb_swap = 1;
-    sensor->hw_flags.bayer = SENSOR_HW_FLAGS_BAYER_GBRG;
+    sensor->hw_flags.bayer = SENSOR_HW_FLAGS_BAYER_BGGR;
```

Comment:
This produces a broken RGB565 image, but a good Grayscale. The old one produces a good RGB565 and bad Grayscale.
![2](https://github.com/user-attachments/assets/495e247f-53e7-4e6e-8a48-19933b03324f)